ember-launch-darkly/package.json

```
@@ -0,0 +1,78 @@
+{
+  "name": "ember-launch-darkly",
+  "version": "3.0.0",
+  "description": "A modern Ember addon to wrap the Launch Darkly service",
+  "keywords": [
+    "ember-addon"
+  ],
+  "repository": "https://github.com/adopted-ember-addons/ember-launch-darkly",
+  "license": "MIT",
+  "author": "The Ember Launch Darkly Team",
+  "directories": {
+    "doc": "doc",
+    "test": "tests"
+  },
+  "files": [
+    "addon-main.js",
+    "dist"
+  ],
+  "scripts": {
+    "build": "rollup --config",
+    "lint": "concurrently 'npm:lint:*(!fix)' --names 'lint:'",
+    "lint:fix": "concurrently 'npm:lint:*:fix' --names 'fix:'",
+    "lint:hbs": "ember-template-lint . --no-error-on-unmatched-pattern",
+    "lint:js": "eslint . --cache",
+    "lint:hbs:fix": "ember-template-lint . --fix --no-error-on-unmatched-pattern",
+    "lint:js:fix": "eslint . --fix",
+    "start": "rollup --config --watch",
+    "test": "echo 'A v2 addon does not have tests, run tests in test-app'",
+    "prepack": "rollup --config"
+  },
+  "dependencies": {
+    "@embroider/addon-shim": "^1.0.0",
+    "launchdarkly-js-client-sdk": "^3.1.1",
+    "tracked-maps-and-sets": "^3.0.2"
+  },
+  "devDependencies": {
+    "@babel/core": "^7.21.3",
+    "@babel/plugin-proposal-class-properties": "^7.18.6",
+    "@babel/plugin-proposal-decorators": "^7.21.0",
+    "@embroider/addon-dev": "^3.0.0",
+    "@rollup/plugin-babel": "^6.0.3",
+    "babel-eslint": "^10.1.0",
+    "concurrently": "^7.6.0",
+    "eslint": "^7.32.0",
+    "eslint-config-prettier": "^8.6.0",
+    "eslint-plugin-ember": "^11.4.7",
+    "eslint-plugin-n": "^15.6.1",
+    "eslint-plugin-prettier": "^4.2.1",
+    "eslint-plugin-qunit": "^7.3.4",
+    "prettier": "^2.8.4",
+    "rollup": "^3.19.1",
+    "rollup-plugin-copy": "3.4.0"
+  },
+  "ember": {
+    "edition": "octane"
+  },
+  "ember-addon": {
+    "version": 2,
+    "type": "addon",
+    "main": "addon-main.js",
+    "app-js": {
+      "./helpers/variation.js": "./dist/_app_/helpers/variation.js"
+    }
+  },
+  "exports": {
+    ".": "./dist/index.js",
+    "./*": {
+      "default": "./dist/*.js"
+    },
+    "./test-support": "./dist/test-support/index.js",
+    "./-sdk": "./dist/-sdk/index.js",
+    "./-sdk/context": "./dist/-sdk/context.js",
+    "./-sdk/identify": "./dist/-sdk/identify.js",
+    "./-sdk/initialize": "./dist/-sdk/initialize.js",
+    "./-sdk/variation": "./dist/-sdk/variation.js",
+    "./addon-main.js": "./addon-main.js"
+  }
```

```suggestion
  },
  "peerDependencies": {
    "ember-source": "^3.28.0 || ^4.0.0"
  },
```

We need to keep the peerDependencies that was in v1, right?

---

Alright, thanks!